oarepo_requests/ui/theme/assets/semantic-ui/less/oarepo_requests_ui/themes/default/elements/button.overrides

```diff
@@ -8,7 +8,11 @@
       color: @white;
     }
     &.publish_draft {
-      color: white;
+      color: @white;
```

Comment:
Can we please make one generalized common class to represent all possible buttons with "publish" or "submit" role? If all such buttons should be white on green?